lib/models/login/login_response_data_model.dart

```diff
@@ -0,0 +1,21 @@
+class LoginResponseDataModel {
```

Comment:
Why do you call this Class LoginResponseDataModel?
is LoginResponseUserDataModel better than LoginResponseDataModel ?

Reply:
??

Reply:
done